examples/component-provider/schema.json

```diff
@@ -0,0 +1,90 @@
+{
+  "name": "go-components",
```

Comment:
Is the `namespace` missing? I see it in the test and in my local example, so won't block on this.